pydis_site/apps/api/tests/test_off_topic_channel_names.py

```diff
@@ -88,10 +88,21 @@ def setUp(self):
         super().setUp()
 
         url = reverse('bot:offtopicchannelname-list', host='api')
-        self.name = "lemonade-shop"
+        self.name = "abcdefghijklmnopqrstuvwxyz-0123456789"
         response = self.client.post(f'{url}?name={self.name}')
         self.assertEqual(response.status_code, 201)
 
+    def test_returns_201_for_unicode_chars(self):
+        url = reverse('bot:offtopicchannelname-list', host='api')
+        names = (
+            '𝖠𝖡𝖢𝖣𝖤𝖥𝖦𝖧𝖨𝖩𝖪𝖫𝖬𝖭𝖮𝖯𝖰𝖱𝖲𝖳𝖴𝖵𝖶𝖷𝖸𝖹',
+            'ǃ？’',
+        )
+
+        for name in names:
+            response = self.client.post(f'{url}?name={name}')
+            self.assertEqual(response.status_code, 201)
+
     def test_name_in_full_list(self):
```

Comment:
this test name is weird. I don't fully understand what it does without the docstring.

Reply:
I agree and it doesn't look like it belong under creation test anyway. In fact, it looks quite similar to `test_returns_name_in_list`

Reply:
Let's just rename it to something that makes sense, then.

Reply:
I was trying to say that this test looks redundant.

Reply:
Aha. Okay well I'd be open to you just scrapping it, too, then.